### ISSUE TITLE
Can't really use LanguageServer in .rmd files

### ISSUE THREAD

**Issue by ChristopheRoyer21** [Ikuyadeu/vscode-R#213]
**Describe the bug**
First of all, when opening a .rmd file, R LanguageServer doesn't start. To do so, I need to create/open a .R file and then it starts. After working a bit on the R code in the .rmd, lsp often stops working and I don't get any more documentation. So far, it looks perfectly fine in .R file

**To Reproduce**
1. Open a .rmd file
2. You see that LanguageServer hasn't started.
2. Open a .R file
3. You see that LanguageServer has started
4. Write some code chunk in the .Rmd
5. LanguageServer quickly stops working

**Expected behavior**
The documentation of the variables and functions in a .Rmd work exactly as it does in a .R file

**Environment (please complete the following information):**
 - OS: Windows 10
 - VSCode Version 1.41.1
 - R Version 3.6.1

**Comment by renkun-ken** [comment]
Currently, the vscode-R extension is only activated by `.R` file. We should make it also activated by `.Rmd` file.

It is strange that language server stops working after you write some code in the `.Rmd` file. Would you like to share more details? What do you observe to have triggered the language server to stop working? You may turn on `r.lsp.debug` in vscode settings, reload or restart vscode, and reproduce the problem. If the same problem occurs, please go to View -> Output -> R Language Server and post all logging of language server. I'll take a closer look at what makes langauge server crash.

**Comment by ChristopheRoyer21** [comment]
I haven't tried it yet, but maybe you could set it to be activated by `.Rnw` files too (knitr and Sweave).

Two thing happen when I'm using lsp with `.Rmd` files. I can't always reproduce both of them.
1. Lsp completely crashes (Exit Error 2)
2. It loads without finding anything, even after several seconds/minutes, for everything I hover. It has not happened to me in `.R` files yet. Maybe it's because of the proximity of the text?

**Comment by renkun-ken** [comment]
>Lsp completely crashes (Exit Error 2)

No logginer at `View -> Output -> R Language Server` at all?

**Comment by ChristopheRoyer21** [comment]
When I look at `View -> Output -> R Language Server` I have en (Exit Error 2) and after that nothing else shows up unless I start again into a `.R` file

**Comment by solarchemist** [comment]
@renkun-ken Sorry to pile on to a stale issue, but this is the only place in this project where Rnw files are mentioned, so I thought I could ask here first. Also, I've rifled through all the forks, and none of them appear to tackle Rnw files either.

Does vscode-R support Rnw files presently? (I'm pretty sure the answer is no).
Would adding support for Rnw files lie within the scope of the project? If so, may I suggest highlighting that target in the README or similar, as a way to draw the attention of possible contributors or users.

*Details of my setup* I've got vscode-R 1.5.2 and R LSP client 0.1.10 on VSCodium 1.51.1, and `languageserver` 0.3.7 (CRAN) installed with R 3.6.2. I opened an Rmd file, and vscode-R kicked right into gear, with boxes on hover, code chunk highlighting and *Run chunk | Run above* commands above each chunk. The log shows `R Language Server (569) started` and nothing else. So it seems my setup is good. 

But all the goodness disappears as soon as you change  the Rmd chunks into Rnw-formatted chunks. \*sad noises\*

**Comment by renkun-ken** [comment]
You are right. vscode-R does not have support for Rnw. Neither does languageserver.

**Comment by solarchemist** [comment]
Oh, so it goes deeper than just vscode-R. Thanks for the clarification!

**Comment by renkun-ken** [comment]
I guess supporting Rnw files is indeed within the scope of vscode-R and language server but obviously there's a lack of community support for such format. Contributions are always welcome.
